packages/react-charts/src/components/ChartLegendTooltip/ChartLegendTooltip.tsx

```diff
@@ -294,7 +293,7 @@ export const ChartLegendTooltip: React.FunctionComponent<ChartLegendTooltipProps
   theme = getTheme(themeColor, themeVariant),
   ...rest
 }: ChartLegendTooltipProps) => {
-  const pointerLength = theme && theme.tooltip ? theme.tooltip.pointerLength : 10;
+  const pointerLength = theme && theme.tooltip ? Number(theme.tooltip.pointerLength) : 10;
```

Comment:
https://github.com/FormidableLabs/victory/commit/439e0ad8de23d1d6ed5580a90f60feaca3f09c4e#diff-1d0ded8dfdfbe48d50e6a99272c7548f81143f224f88b6528023cc87a32e9a40R462
Here `pointerLength` is no longer `number` but `NumberOrCallback`. So we need to cast it to `number` type.

Reply:
In that case, we may want to do something more like this:

const pointerLength = theme.tooltip.pointerLength === 'function' ? theme.tooltip.pointerLength() : Number(theme.tooltip.pointerLength);

Reply:
Victory also has the helper function below we could use:

```
export { default as Helpers } from "./victory-util/helpers";
```

```
function evaluateProp(prop, props) {
  return isFunction(prop) ? prop(props) : prop;
}
```

Reply:
Actually, the `isFunction` comes from lodash, which is also available